fineract-provider/src/main/resources/sql/migrations/core_db/V312__Creditbureau_configuration.sql

```diff
@@ -0,0 +1,61 @@
+
+
+CREATE TABLE `m_creditbureau` (
+  `id` bigint(20) NOT NULL AUTO_INCREMENT,
+  `name` varchar(100) NOT NULL,
+  `product` varchar(100) NOT NULL,
+  `country` varchar(100) NOT NULL,
+  `implementationKey` varchar(100) NOT NULL,
+  PRIMARY KEY (`id`),
+  UNIQUE KEY `unique impl` (`name`,`product`,`country`,`implementationKey`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+
+CREATE TABLE `m_organisation_creditbureau` (
```

Comment:
What is the use of this table? Under same tenant can we have more than one organization alias?